work-process/user-stories.md

```diff
@@ -9,6 +9,14 @@
   *  they assign people from whom they want to receive feedback for better definition of the User Story.
   * Once these people post their feedback, they unassign themselves from the User Story.
 - Once the Scrum Master and Product Owner agree that the User Story is well defined, they add ``✋ [vote]`` prefix to the title of the User Story. At that point, Scrum Master and/or Product Owner assign people from whom they want to receive User Story Points estimation. With this, online poker planning begins.
+-  Assignees add their estimates in a comment with the following structure:
+```
+<details>
+  <summary>Click to see estimate</summary>
```

Comment:
I would change this to `Story Point`

Reply:
The correct terminology is giving estimation and not giving story points.
I don't see why we don't want to add additional information that user should click to see the estimation. Because that arrow can be an icon and some users might be confused.

Reply:
Ok `Story estimate`

We don't want users to click the estimate so remove the idea of clicking on it.